docker-compose/docker-compose.yaml

```diff
@@ -1,23 +1,22 @@
-version: "3"
+version: "3.8"
 
 services:
   iroha:
-    image: hyperledger/iroha2:lts@sha256:4072d09bee376c8fcbc586d35ea25624738694ce2c37aa0db60c9998c613048b
+    image: hyperledger/iroha2:stable
```

Comment:
we should include a hash too, though this build should also become an `lts` and we could wait for that